src/components/NavBar/index.js

```diff
@@ -33,9 +33,31 @@ class NavBar extends React.Component {
                             </NavLink>
                         </li>
                         <li>
-                            <NavLink to="/addPlaces/" activeClassName="active">
-                                AddPlaces
+                            <NavLink
+                                to="/addNew/"
+                                className="disabled"
+                                activeClassName="active"
+                            >
```

Comment:
here you can use onClick={(ev) => ev.preventDefault()} in order to prevent navigation to the non existing "/addNew/" page